src/cpp/session/SessionAsyncRProcess.cpp

```diff
@@ -80,8 +80,16 @@ void AsyncRProcess::start(const char* rCommand,
    std::vector<std::string> args;
 
    if (rOptions & R_PROCESS_VANILLA)
+   {
       args.push_back("--vanilla");
-
+   }
+   else if (rOptions & R_PROCESS_VANILLA_USER)
+   {
+      args.push_back("--no-save");
```

Comment:
What about Renviron files? `R --help` has:

```
$ R --help | grep no-environ
  --no-environ          Don't read the site and user environment files
```

Do we want to:

1. Read all Renviron files;
2. Read only the site Renviron file,
3. Read none of the Renviron files (since we're inheriting environment variables anyway)

Reply:
There are two site Renviron files: `R_HOME/etc/Renviron` and `R_HOME/etc/Renviron.site`. `--no-environ` blocks `R_HOME/etc/Renviron.site` and the user Renviron, but `R_HOME/etc/Renviron` is always read and cannot be blocked. So we want `--no-environ` because it blocks the user Renviron, but still the core environment variables are always exported.

On the other hand, there's only one site Rprofile, so we want just `--no-init-file`, as you just did, so LGTM. Many thanks for this! :)